src/fairchem/core/trainers/base_trainer.py

```diff
@@ -294,6 +294,8 @@ def load_datasets(self) -> None:
                 self.train_dataset,
                 self.train_sampler,
             )
+        elif self.config.get("dataset", None) is None:
+            self.config["dataset"] = {}
```

Comment:
@zulissimeta why are empty dictionaries for datasets needed? are the tutorials broken otherwise?

Reply:
The code in the other datasets try to copy from train_dataset by default ("use_train_settings"). If train_dataset isn't present, this breaks. Now, if train isn't present it just starts with a blank dictionary. 

It feels strange to me that the default path is to try this and then cause errors if train is not present, as there's nothing about the config that suggests val/test/relax are about to try to pull settings from train.

Reply:
I see, that is a good point!